src/Select/select/Item.ts

```diff
@@ -0,0 +1,17 @@
+export type OptionProps = Omit<
+  React.DetailedHTMLProps<
+    React.OptionHTMLAttributes<HTMLOptionElement>,
+    HTMLOptionElement
+  >,
+  "children"
+> & { children: string };
```

Comment:
Hmm does this mean we cant have any icons or formatting inside an option? that seems like it could cause some problems down the line?

Reply:
@Jephuff That's exactly what this means. Remember, this is intended to replace the functionality of a `<select />`. If we want to add more functionality down the line; we totally can open this up if we want to somehow customize the select menu.

This has no effect on using the already existing `List` and `ListItem`s; only `Select`.